explanations/maintainer-missing.md

```diff
@@ -0,0 +1,3 @@
+All packages ought to have a maintainer.
+
```

Comment:
I remember seeing something describing what is expected of maintainers. Would be nice to link it here but cannot find it now.

Reply:
Maybe it was a discussion on discourse? That would be useful.